concepts/team-changenotification-teams.md

```diff
@@ -0,0 +1,211 @@
+  "encryptionCertificateId": "{customId}",
+  "expirationDateTime": "2019-09-19T11:00:00.0000000Z",
+  "clientState": "{secretClientState}"
+}
+```
+
+### Subscribe to changes in a particular team
+
+
+To get change notifications for any change in teams' property of a specific team, subscribe to `/teams/{id}`. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification.
+
+#### Permissions
+
+|Permission type      | Permissions (from least to most privileged)              | Supported versions |
+|:--------------------|:---------------------------------------------------------|:-------------------|
+|Delegated (work or school account) | Not supported. | Not supported. |
+|Delegated (personal Microsoft account) | Not supported.    | Not supported. |
+|Application | Team.ReadBasic ,TeamSettings.Read    | beta, v1.0 |
+
+#### Example
+
+```http
+POST https://graph.microsoft.com/beta/subscriptions
+Content-Type: application/json
+
+{
+  "changeType": "deleted,updated",
+  "notificationUrl": "https://webhook.azurewebsites.net/api/resourceNotifications",
+  "resource": "/teams/{id}",
+  "includeResourceData": true,
+  "encryptionCertificate": "{base64encodedCertificate}",
+  "encryptionCertificateId": "{customId}",
+  "expirationDateTime": "2019-09-19T11:00:00.0000000Z",
+  "clientState": "{secretClientState}"
+}
+```
+
+
+## Notification payloads
+
+Depending on your subscription, you can either get the notification with resource data, or without it. Subscribing with resource data allows you to get the message payload along with the notification, removing the need to call back and get the content.
+
+### Notifications with resource data
+
+For notifications with resource data, the payload looks for any change in team's property looks like the following.
+
+```json
+{
+    "value": [{
+        "subscriptionId": "10493aa0-4d29-4df5-bc0c-ef742cc6cd7f",
+        "changeType": "created",
+        "clientState": "<<--SpecifiedClientState-->>",
+        "subscriptionExpirationDateTime": "2021-02-02T10:30:34.9097561-08:00",
+        "resource": "teams('fb82c19a-0f6d-41ed-90f0-cbb29a476ede')",
+        "resourceData": {
+            "id": "1612289765949",
+            "@odata.type": "#Microsoft.Graph.Teams",
+            "@odata.id": "teams('fb82c19a-0f6d-41ed-90f0-cbb29a476ede')"
+        },
+        "encryptedContent": {
+            "data": "<<--EncryptedContent-->",
+            "dataKey": "<<--EnryptedDataKeyUsedForEncryptingContent-->>",
+            "encryptionCertificateId": "<<--IdOfTheCertificateUsedForEncryptingDataKey-->>",
+            "encryptionCertificateThumbprint": "<<--ThumbprintOfTheCertificateUsedForEncryptingDataKey-->>"
+        },
+        "tenantId": "<<--TenantForWhichNotificationWasSent-->>"
+    }],
+    "validationTokens": ["<<--ValidationTokens-->>"]
+}
+```
+  
+  
+For notifications with resource data, the payload looks for any change in team's property looks like the following.
```

Comment:
same content twice ?
For notifications with resource data, the payload looks for any change in team's property looks like the following.